apps/docs/src/app/core/component-docs/popover/examples/popover-complex-example/popover-complex-example.component.html

```diff
@@ -17,16 +17,12 @@
             <button fd-button label="With Header"></button>
         </fd-popover-control>
         <fd-popover-body>
-            <div fd-popover-body-header>
-                <div fd-bar [cozy]="true">
-                    <div fd-bar-left>
-                        <fd-bar-element>Cozy Header</fd-bar-element>
-                    </div>
-                </div>
-            </div>
```

Comment:
Why do you remove this?

Reply:
on using the fd-bar-element the focus in not directed to the header part so the change is required to migrate to list header.

Reply:
I don't understand your answer. Please provide more info. Thank you

Reply:
The header in this case is part of the Popover, it's not part of the List. You should be able to have a header without a List:
https://fundamental-styles.netlify.app/?path=/docs/components-popover--alignment

<img width="1418" alt="Screen Shot 2021-07-05 at 3 30 55 PM" src="https://user-images.githubusercontent.com/39598672/124512290-01d55580-dda6-11eb-8c29-3b2e37a0bc30.png">